canonical/ubuntu.com
# /certified refresh WD-741

## Done

- Refreshed /certified according to the [design](https://www.figma.com/file/Mpyhs1ZUsyeKxltxOskYp4/Certified-refresh?node-id=93%3A12567&t=7xGzlbmGYzHaShYX-1)


## QA

- Visit https://ubuntu-com-12353.demos.haus/certified
- See that it matches the design and the [copy doc](https://docs.google.com/document/d/1Cq9RLNjSbX83WB5fFKYCFRvo_1JEfMydMecqrVeKBP8/edit)

## Issue / Card

Fixes [WD-741](https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/932?modal=detail&selectedIssue=WD-741)
